aries_vcx/tests/utils/test_agent.rs

```diff
@@ -81,3 +83,38 @@ pub async fn create_test_agent(
 > {
     create_test_agent_from_seed(&generate_random_seed(), genesis_file_path).await
 }
+
+pub async fn create_test_endorser<LW, W>(
```

Comment:
nitpick: let's call this `create_test_agent_endorser` for consistency

Alternatively we could drop `_test_` from all of these methods keeping it just `create_agent_<foobar>`